src/styles/theme.scss

```diff
@@ -1,20 +1,21 @@
 $text-red: #f53535;
-$text-black: #000;
-$text-white: #fff;
+$text-black: #000000;
+$text-white: #ffffff;
 
 $primary: #00f4d7;
 $primary-blue: #3800d6;
 $primary-blue-light: #f5f2fd;
 $primary-blue-dark: #0d0033;
 $white: #ffffff;
-$black: #000;
+$black: #000000;
 $error: #fb5a5a;
-$black80: #333333;
+$black80: #{$black}cc;
```

Comment:
can we not use alpha for colors? Also colors with alpha should be labeled as such eg `$blue40T` (but again, lets not use alpha where we can avoid it)